.codex/skills/swcc/agents/zhongjian.md

```
@@ -0,0 +1,106 @@
+[重复以上结构...]
+
+## 实施建议
+
+### 优先级排序
+1. [最重要/风险最低的改动先做]
+2. [次要改动]
+3. [可选的改进，如果时间允许]
+
+### 回退方案
+[如果折中方案某部分出问题，如何安全回退]
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)